sdk/template/ci.yml

```diff
@@ -0,0 +1,42 @@
+resources:
+  repositories:
+    - repository: azure-sdk-build-tools
```

Comment:
Is this still necessary?

Reply:
hopefully not.

Reply:
Yes, it's still used for Publishing and GPG signing